docs/API/V1/contents.md

```diff
@@ -1,14 +1,14 @@
 # `GET /api/v1/datasets/contents/<dataset><path>`
 
-This API returns an `application/json` document describing the contents of the referenced relative `<path>` within the `<dataset>` tarball representation.
+This API returns an `application/json` document describing a `<path>` within the `<dataset>` tarball representation.
```

Comment:
I'm not quite sure what it means to "describe a path"...perhaps this API describes an "item" found at the location indicated by `<path>`?  This comment applies to line 36, as well.